doc/guide/cockpit-dbus.xml

```diff
@@ -86,6 +86,13 @@
         <listitem><para>A javascript plain object with the <code>"t"</code> property set to a DBus type string,
             and the <code>"v"</code> property set to a value.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term><code>HANDLE 'h'</code></term>
+        <listitem><para>A javascript object that describes a cockpit channel which represents the
+            passed file descriptor. Pass it to <link linkend="cockpit-channels-channel">
```

Comment:
I left it intentionally opaque, because the channel is openend via an internal id which is useless anyway. I'll add that this is always a stream channel.